tests/kernel/gen_isr_table/testcase.yaml

```diff
@@ -8,7 +8,6 @@ tests:
       - CONFIG_NULL_POINTER_EXCEPTION_DETECTION_NONE=y
   arch.interrupt.gen_isr_table.arm_baseline.linker_generator:
     platform_allow: qemu_cortex_m3
-    platform_allow: bbc_microbit atsamr21_xpro nrf51dk_nrf51422 nucleo_g071rb qemu_cortex_m0
```

Comment:
Shouldn't these be combined, instead of removed? Ditto for above.

Reply:
Nope, this was just a copy/paste problem, the linker_generator should only be verified with qemu_cortex_m3, and nothing else.

The one above IMO should not have specific platforms "allowed" I do not see any reason why those should be the only platforms where this has to run, I think using qemu_cortex_m3 should be enough here or it should be open for all.